src/test/java/org/mule/extension/sftp/SftpWriteTestCase.java

```diff
@@ -47,6 +50,8 @@ protected String getConfigFile() {
 
   @Test
   public void writeOnAPathWithColon() throws Exception {
+    //TODO: Remove assumption once MULE-15733 get fixed.
```

Comment:
Add this as a comment on the JIRA ticket.

Reply:
Good idea: Done in 👉 [link-to-comment](https://www.mulesoft.org/jira/browse/MULE-15733?focusedCommentId=257588&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-257588])